src/main/java/com/hillel/items_exchange/security/jwt/JwtUserFactory.java

```diff
@@ -23,12 +23,21 @@ public static JwtUser create(User user) {
                 user.getEmail(),
                 user.getPassword(),
                 mapToGrantedAuthorities(user.getRole()),
-                user.getStatus().equals(Status.ACTIVE),
+                isUserEnabled(user),
                 user.getUpdated()
         );
     }
 
     private static List<GrantedAuthority> mapToGrantedAuthorities(Role userRole) {
         return Collections.singletonList(new SimpleGrantedAuthority(userRole.getName()));
     }
+
+    private static boolean isUserEnabled(User user) {
+        boolean isUserEnabled = false;
+        if (user.getStatus().equals(Status.ACTIVE) || user.getStatus().equals(Status.DELETED)) {
+            isUserEnabled = true;
+        }
+
+        return isUserEnabled;
+    }
```

Comment:
It can be simplified:

```
Status userStatus = user.getStatus();
 
return userStatus.equals(Status.ACTIVE) || userStatus.equals(Status.DELETED);
```